src/App.ts

```diff
@@ -38,7 +38,11 @@ export async function startApp(config: Configuration): Promise<http.Server> {
     await PPTPortClient.createPagamentiTelematiciPspNodoClient({
       endpoint: `${config.PAGOPA.HOST}:${config.PAGOPA.PORT}${
         config.PAGOPA.WS_SERVICES.PAGAMENTI
-      }`
+      }`,
+      // Custom HTTP headers to be sent on WSDL requests
+      wsdl_headers: {
```

Comment:
make it optional, like

```ts
wsdl_headers: config.PAGOPA.HOST_HEADER ? { Host: config.PAGOPA.HOST_HEADER } : { }
```

Reply:
It's optional yet. Check the type declaration at line 106:
t.partial({
    HOST_HEADER: t.string
  })

Reply:
yes but when HOST_HEADER is not defined you're still passing it to the soap library as the Host header!!!

Reply:
Nope. Client Header is set just if it's defined:

https://github.com/teamdigitale/italia-pagopa-proxy/blob/Add_PagoPa_Host_Header/src/utils/SoapUtils.ts#L22